docs/admin/support.rst

```diff
@@ -53,37 +53,37 @@ Discover Weblate
 
 .. versionadded:: 4.5.2
 
-Discover Weblate is an opt-in service that makes it easier for users to find
-Weblate servers and communities. Users can browse registered services on
-<https://weblate.org/discover/>, and find there projects to contribute.
+*Discover* is an opt-in service making it easier for users to find
+other Weblate sites and communities.
+Users can browse registered sites and find projects to contribute to on
+https://weblate.org/discover/.
```

Comment:
There was some escaping for URLs ending in dots in another PR. Maybe needed here?

Reply:
https://weblate--5761.org.readthedocs.build/en/5761/ shows how this gets rendered. This is available for all pull request, look for Read the docs in checks